app/views/hosts/_selected_hosts.html.erb

```diff
@@ -1,4 +1,6 @@
 <div class="row">
+  <%= alert(:class => 'alert-info hide', :close => true, :header => '',
```

Comment:
@ohadlevy For which part? The selection of all items or that all items are changing owners?